scripts/config/Makefile

```diff
@@ -33,11 +37,22 @@ clean-files	:= zconf.tab.c zconf.lex.c zconf.hash.c
 # Remove qconf junk files
 clean-files	+= $(qconf-cxxobjs) qconf.moc .tmp_qtcheck qconf
 
-all: conf mconf
+all: conf mconf nconf
+
+$(obj)/%conf-cfg: $(src)/%conf-cfg.sh FORCE
+        $(call filechk,conf_cfg)
+
+HOSTLDLIBS_nconf = $(shell eval `./nconf-cfg.sh` && echo $$libs)
+HOSTCFLAGS_nconf = $(shell eval `./nconf-cfg.sh` && echo $$cflags)
```

Comment:
I'd prefer to keep the minimum difference from linux kconfig. Why it's integrated into this Makefile for QT?